test/JDBC/expected/constraint_table_usage-vu-verify.out

```diff
@@ -0,0 +1,48 @@
+use constraint_table_usage_prepare_db;
+go
+
+select * from information_schema.constraint_table_usage where table_name like 'constraint_table_usage_tb%';
```

Comment:
The table names are starting with `constraint_table_usage_prepare_tb%`. Please modify it.